packages/@okta/vuepress-site/guides/sign-into-spa/section03/react/config.md

```diff
@@ -1,2 +1,71 @@
-In your application's `App.js` file, import the following objects and pass in your configuration:
-{code}
\ No newline at end of file
+Configure a Security component in your top-level component, 'App.jsx'. The Security component will create a child context which exposes the Auth object to child components.
+
+```javascript
+
+
+import React, { Component } from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Security } from '@okta/okta-react';
+
+
+const config = {
+  clientId: '{clientId}',
+  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
+  redirectUri: 'http://localhost:8080/implicit/callback',
+};
+
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <Security {...config} >
+          { /* App routes go here */ }
+        </Security>
+      </Router>
+    );
+  }
+}
+
+export default App;
+```
+
+If you find that your application needs to interact with the Auth object outside of a child component context, you can create the Auth object directly and pass it to the Security component as shown below.
+
+Note that Auth has a dependency on the 'history' npm module. You should create this history object and pass it as a property to the Auth construxtor object as shown:
```

Comment:
Note that Auth has a dependency on the `history` npm module. You should create this history object and pass it as a property to the Auth constructor object: